firefox-ios/firefox-ios-tests/Tests/ClientTests/ContentBlockerTests.swift

```diff
@@ -0,0 +1,30 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import XCTest
+@testable import Client
+
+final class ContentBlockerTests: XCTestCase {
+    override func setUp() {
+        super.setUp()
+        ensureAllRulesAreRemovedFromStore()
+    }
+
+    private func ensureAllRulesAreRemovedFromStore() {
+        let expectation = XCTestExpectation()
+        ContentBlocker.shared.removeAllRulesInStore {
+            expectation.fulfill()
+        }
+        wait(for: [expectation])
+    }
+
+    func testCompileListsNotInStore_callsCompletionHandlerSuccessfully() {
+        ensureAllRulesAreRemovedFromStore()
```

Comment:
If it's in the setup we can remove it from the top of the func too 👀 
```suggestion
```

Reply:
Sorry @lmarceau i forgot to remove it, my bad. Tomorrow I'm gonna solve it.

Reply:
No problem! 🙏